LEGO1/infocenterstate.h

```diff
@@ -23,6 +25,36 @@ class InfocenterState : public LegoState
   {
     return !strcmp(name, InfocenterState::ClassName()) || LegoState::IsA(name);
   }
+
+  inline MxU32 GetInfocenterBufferElement(int p_index) { return m_buffer[p_index]; }
+
+private:
+  // Size: 0xC
+  struct SomeStruct
+  {
+    undefined4 unk1;
+    undefined2 unk2;
+    undefined2 unk3;
+    undefined2 unk4;
+  };
+
+  undefined2 unk1;
```

Comment:
`unks` should be numbered by their offset so they can more easily be referred to in asm/pseudocode.